run.py

```diff
@@ -189,17 +227,51 @@ def download_playlist(spotify_playlist_id, folder_name):
                 if(not isinstance(e, ConfigException)):
                     print('Please report this at https://github.com/couldbejake/spotify2mp3' + bcolors.ENDC)
                     quit()
+                else:
+                    f = open('failed_log.txt', 'a')
+                    f.write(search_query + '\n')
+                    f.write(str(e))
+                    f.write('\n')
+                    f.write(traceback.format_exc())
+                    f.write('\n')
+                    f.close()
+                    print(f"{bcolors.WARNING}Failed to convert {search_query} due to config error.{bcolors.ENDC}")
+                    failed_downloads += 1
+                    failed_song_names = failed_song_names + "\t• " + song_name + " - " + artist + f" | Fail reason: {e}" + "\n"
+    print(f"{bcolors.OKGREEN}Successfully downloaded {len(songs) - failed_downloads - skipped_songs}/{len(songs)} songs ({skipped_songs} skipped) to {folder_name}{bcolors.ENDC}\n")
+    if failed_downloads >= FAILURE_THRESHOLD:
+        if "y" in input(f"\n\nThere were more than {FAILURE_THRESHOLD} failed downloads:\n{failed_song_names} \n\nWould you like to retry with minimum view count halfed({MIN_VIEW_COUNT//2})? (y/n) "):
```

Comment:
The download should be automatic, having a user prompt may delay downloads.

I will update this in a future commit.

Reply:
I've just noticed that this check is done at the end, which is fine.

Thank you